servicetalk-http-utils/src/main/java/io/servicetalk/http/utils/TimeoutFromRequest.java

```diff
@@ -51,4 +51,28 @@ default HttpExecutionStrategy influenceStrategy(HttpExecutionStrategy strategy)
         // The effects of apply() are unknown so the default strategy should be used.
         return HttpExecutionStrategyInfluencer.defaultStreamingInfluencer().influenceStrategy(strategy);
     }
+
+    /**
+     * Returns a function which returns the provided default duration as the timeout duration to be used for any
+     * request.
+     *
+     * @param duration timeout duration or null for no timeout
+     * @return a function to produce a timeout using specified duration
+     */
+    static TimeoutFromRequest simpleDurationTimeout(@Nullable Duration duration) {
```

Comment:
Any specific reason for making it public now (all members of a public interface are public)? The function is trivial and does not provide much value when constructors just take `Duration`. Let's move it back as a pkg-private static method of a class.